changelog.md

```diff
@@ -20,6 +20,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Changed
 - [[#3062] Improve Compilation Error Messages to be more specific](https://github.com/ballerina-platform/ballerina-standard-library/issues/3062)
+- [[#2848] Only One Kind of Errors Return From a Invalid GraphQL Request](https://github.com/ballerina-platform/ballerina-standard-library/issues/2848)
```

Comment:
```suggestion
- [[#2848] Only One Kind of Errors Return From an Invalid GraphQL Request](https://github.com/ballerina-platform/ballerina-standard-library/issues/2848)
```

Reply:
Shall we update the previous entry as well? Seems to have mixed up the title case.

Reply:
Updated with AP style